db/seeds.rb

```diff
@@ -1,7 +1,81 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
+name = Faker::Games::Pokemon.name
+email = Faker::Internet.email
+User.create!(
+             name: name,
+             email: email,
+             password: '00000000',
+             admin: 'true',
```

Comment:
boolean型の値を文字列にする必要があるかどうか、確認をお願いします。
```suggestion
             admin: true,
```